docs/user/exams/live_statistics.rst

```diff
@@ -0,0 +1,129 @@
+***************
+Live Statistics
+***************
+
```

Comment:
> Additional question: Is there a way to globally disable the feature as an admin in the server configuration file? This could be potentially important to satisfy the legal department at the university.

Yes, via the global feature toggle :+1:

> I think it would be nice to have just a short bullet-point list at the top with an overview which data could be collected by this feature. That way instructors get an easy understanding if they would benefit from that feature and if they are even allowed to enable it (GDPR).

Sure, thats a good idea

Reply:
No, that's correct :+1: